### ISSUE TITLE
Parallel fork doesn't execute tests which are under parallelize_me! execution regime

### ISSUE THREAD

**Issue by bbozo** [jeremyevans/minitest-parallel_fork#1]
Simple workaround is to:

``` ruby

class MiniTest::Spec
  def self.parallelize_me!
    # do nothing in parallel execution mode
  end
end
```

**Comment by jeremyevans** [comment]
This is due to Minitest::Parallel overriding run_one_method for the test suite.  I have a fix locally, as soon as I've fully tested it, I'll push it up here and release a new version.

**Comment by jeremyevans** [comment]
Fixed by 60cc5790c702c147d7d77fa190df1a6b725a046a

**Comment by bbozo** [comment]
<3 :)
